test/application/support/setup.js

```diff
@@ -0,0 +1,61 @@
+const { Before, BeforeAll, AfterAll, After } = require("@cucumber/cucumber");
+const { chromium } = require("playwright");
+const axios = require("axios");
+
+const users = {
```

Comment:
These users dont match the users in test/application/support/env.js. Would this be a problem/change needed going forward?

Reply:
Good catch - I've made the two files consistent,  but as you can see they're not being used properly yet.

At the moment they're used to mentally separate out the participants in the test - I'm trying to take a few pointers from https://cucumber.io/docs/bdd/better-gherkin/

Once these can integrate with the backend and/or the stub, then I'm intending on tweaking the code to use the values given.